### PR TITLE
Main

### PR DESCRIPTION
**Pull request by marcoooo**
## Description

Backport latest updates from main onto current release branch (GIFTS / Alphafold related)

## Use case

Alphafold run for 112

## Benefits

Best code

## Possible Drawbacks

N/A

## Testing

- [ ] Have you added/modified unit tests to test the changes?
- [ ] If so, do the tests pass?
- [ ] Have you run the entire test suite and no regression was detected?
- [ ] TravisCI passed on your branch

Dependencies
------------

> If applicable, define what code dependencies were added and/or updated.

### CODE REVIEW

File: modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/EarlyDumps_conf.pm
Comment (EbiArnie) [inline modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/EarlyDumps_conf.pm:1]: This file is unrelated, I think. Should not be part of this backport.

**Review by EbiArnie — CHANGES_REQUESTED** [review]
The Pipeconf file seems unrelated. Is this needed to run Alphafold?
If not, please put into separate PR.

**Review by EbiArnie — APPROVED** [review]
OK

**Review by dpopleton — APPROVED** [review]
